_posts/2022-04-25-tardigrade-launch.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: post
+title:  "ETL in Trino speeds with project Tardigrade: now ready for early users"
+author: Andrii Rosa, Brian Zhan, Lukasz Osipiuk, Zebing Lin
+excerpt_separator: <!--more-->
+---
+
+After 6 months of challenging work, we’ve launched Project Tardigrade. It is 
```

Comment:
> we’ve launched Project Tardigrade

It should say "we've completed Project Tardigrade".

Reply:
Rephrased to `After six months of challenging work on Project Tardigrade, we are ready to launch.` based on @mosabua suggestion